tests/phpunit/integration/DAApiTest.php

```diff
@@ -50,6 +55,125 @@ public function testGetServerInfo(): void {
 		$this->assertSame( ServerInfo::DA_API_VERSION, $data['api_version'] );
 	}
 
+	/**
+	 * @covers \DataAccounting\API\GetHashChainInfoHandler
+	 */
+	public function testHashChainInfoHandler(): void {
+		// Testing the case when the rev_id is found.
+		$services = [
```

Comment:
I'm merging, but also wondering if it is possible to deduplicate these similar steps when the rev_id is found, permissions denied, and testing expected fields in the json. Not a priority though.

Reply:
Made #253.